index.html

```diff
@@ -208,6 +225,29 @@ <h2> Deliverables </h2>
         <div id="normative">
           <h3> Normative Specifications </h3>
           <dl class='todo'>
+            <dt class="spec"><a href="https://www.w3.org/TR/ttml3/">Timed Text
+                Markup Language 3 (TTML3)<br>
+              </a></dt>
+            <dd>This collection of specifications defines a content type that
+              represents timed text media for the purpose of interchange among authoring, distribution and playback systems. 
+              Timed text is textual
+              information that is intrinsically or extrinsically associated with
+              timing information. </dd>
+            <dd>
+              <p class="draft-status"><b>Draft state:</b> <i><a href="https://w3c.github.io/ttml3/">Editor's
+                    Draft</a></i></p>
+              <p class="milestone"><b>Expected completion: [Q4-2019]</b> </p>
```

Comment:
I'm not comfortable saying Q4 is "expected". While that may be our goal, I would be more comfortable saying "Q3-2020" for expected.

Reply:
@skynavga to do that would be a divergence from the working mode we agreed to adopt, in which features that we could not get through the process during this year would be deferred. If we do not expect to publish until Q3 2020 then I question whether we should even start. There was a lot of pressure caused by how long TTML2 took, and I think there was agreement to do things differently next time.

Reply:
@nigelmegitt now that you have updated the deliverable info for TTML3 to one spec, it is possible to meet that schedule; however, I have doubts that all the issues/requirements that we have delegated to modules will complete